easybuild/easyconfigs/g/GenotypeHarmonizer/GenotypeHarmonizer-1.4.14-Java-1.7.0_80.eb

```diff
@@ -0,0 +1,28 @@
+
+name = 'GenotypeHarmonizer'
+version = '1.4.14'
+
+homepage = 'https://github.com/molgenis/systemsgenetics/wiki/Genotype-Harmonizer'
+description = """The Genotype Harmonizer is an easy to use command-line tool that allows harmonization of genotype data stored using different file formats with different and potentially unknown strands."""
```

Comment:
per line, I didn't shorten the description, just broke it up in multiple lines, see above